### ISSUE TITLE
bug in code when substitute more then one line

### ISSUE THREAD

**Issue by joakimnyden** [nautobot/nornir-nautobot#11]
Problem occure when we have more then one line to substitute from the running config.
what happends is that for every extra line we have defined in substitute-line we also get an extra copy of the config-line.

https://github.com/nautobot/nornir-nautobot/blob/61172c6ec4ea6aecac1e8b0c4da023eb40381b40/nornir_nautobot/plugins/tasks/dispatcher/default.py#L291-L302

example issue when having 3 lines defined to match against:
``` shell
Building configuration...
Building configuration...
Building configuration...



Current configuration : 76809 bytes
Current configuration : 76809 bytes
Current configuration : 76809 bytes
```

suggestion:
```python
       # Loop through the config lines
        for line in config.split("\n"):
            # Loop through the replacement list on each line in the config
            match = False
            for item in substitute_lines:
                if re.match(pattern=fr"{item['regex_search']}", string=line):
                    line = line.replace(
                        re.match(pattern=fr"{item['regex_search']}", string=line).groups()[0], item["regex_replacement"]
                    )
                    match = True
                if match is True:
                    new_config += line.rstrip() + "\n"
                    break
            if match is False:
                new_config += line.rstrip() + "\n"
        config = new_config
```

**Comment by itdependsnetworks** [comment]
I think I overcomplicated this, I think I just need:

```
for item in substitute_lines:
    config = re.sub(fr'{regex}', '', config, flags=re.MULTILINE)
```

**Comment by jvanderaa** [comment]
@itdependsnetworks is this going to need an update?

**Comment by jvanderaa** [comment]
Closed out with the release of 2.0.0
